react-native-in-app-payments-quickstart/app/components/CardsOnFileCardView.tsx

```diff
@@ -29,21 +29,21 @@ CardsOnFileCardView.propTypes = {
   onSelectCardOnFile: PropTypes.func.isRequired,
 };
 
-function showConfirmation(cardOnFile, onConfirm) {
+function showConfirmation(cardOnFile: any, onConfirm: any) {
   Alert.alert('Confirm',
     `Purchase a cookie for $1 using your ${cardOnFile.card_brand} ${cardOnFile.last_4} card`, [
-      {
-        text: 'Cancel',
-        style: 'cancel',
-      },
-      {
-        text: 'Purchase',
-        onPress: onConfirm,
-      },
-    ]);
+    {
+      text: 'Cancel',
+      style: 'cancel',
+    },
+    {
+      text: 'Purchase',
+      onPress: onConfirm,
+    },
+  ]);
 }
 
-export default function CardsOnFileCardView({ cardOnFile, onSelectCardOnFile }) {
+export default function CardsOnFileCardView({ cardOnFile }: { cardOnFile: any }, { onSelectCardOnFile }: { onSelectCardOnFile: any }) {
```

Comment:
please use a concrete type instead of `any`

Reply:
Resolved it.